scripts/run_agent.sh

```diff
@@ -67,14 +67,26 @@ if [ "$AGENT_ENVIRONMENT" = "preview" ] && [ -n "$PR_NUMBER" ]; then
         
         cd "$AGENT_DIR"
         cp ../.env .
-        npm install
```

Comment:
Revert all changes to this run_agent script. We can assume that it's on a valid machine